src/js/letters/components/Address.jsx

```diff
@@ -13,26 +14,19 @@ import { isNotBlank, isBlankAddress, isValidUSZipCode } from '../../common/utils
  * No validation is provided through a currently stubbed isAddressValid function.
  */
 class Address extends React.Component {
-  constructor() {
-    super();
-
-    this.isValidAddressField = this.isValidAddressField.bind(this);
-    this.isValidPostalCode = this.isValidPostalCode.bind(this);
-  }
-
   componentWillMount() {
     this.id = _.uniqueId('address-input-');
   }
 
-  isValidAddressField(field) {
+  isValidAddressField = (field) => {
```

Comment:
ES6's arrow functions don't have their own 'this', so just use the 'this' of the calling function. So basically you don't have to bind the method in the constructor.

Reply:
Yup, exactly what @markgreenburg said. I just think it's nicer than using the constructor to bind `this` to all the functions. It's really not super important in this case, but when you pass functions as props to children, things get confusing since `this` is perhaps not what you expect. I just like to be consistent with it when I think about it.